### DISCUSSION TITLE
Can accelerated page movement be disabled when using the mouse wheel while it is over scroll bar?

### DISCUSSION THREAD

**Discussion by githwithit** [sumatrapdfreader/sumatrapdf#3180] [Q&A]
When I'm viewing an epub file I use the wheel on my mouse to move the page up and down about a line at a time per mouse wheel tick.

But, if my mouse cursor has inadvertently moved to the right and is over the right hand scroll bar, 1x mouse wheel tick moves 12-13 lines at a time.

Can this change in page movement while mouse wheeling over the right hand scroll bar be disabled?

I would like the 1line per mouse wheel tick to be movement no matter where my mouse cursor is over the program.

**Comment by GitHubRulesOK** [comment]
hmm 
no setting seems to have any reduction affect other than smaller window height OR scroll bar off (right click)
